demos/benchmarking/docker-jans-loadtesting-jmeter/scripts/add_users_rdbm.py

```
@@ -110,18 +114,18 @@ def load_users(interval):
         inum = str(uuid.uuid4()).upper()
         name = user_id_prefix + str(int(start))
         sn = user_id_prefix + '_sn' + str(int(start))
-        dn = 'inum={},ou=people,o=jans'.format(inum)
+        dn = 'inum={},{}'.format(inum, user_parent_dn)
```

Ruff (0.16.1)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)